README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+---
```

Comment:
A small technical note, but you don't need this in READMEs because they're plain markdown. Rmd files, which we'll be using for the rest of class, will have this header included automatically.